tokens/tokens_handlers_test.go

```diff
@@ -30,18 +29,19 @@ var (
 	}
 )
 
-func expireZeroValidTokenOp() TokenOptions {
+func expiredValidTokenOp() TokenOptions {
 	op := validTokenOp
-	op.Duration = 0
+	// This will set the expiry to 1 second ago
+	op.Duration = -1
 	return op
 }
 
 func TestExpiredLoginToken(t *testing.T) {
-	fakeToken, err := GenerateLoginToken(expireZeroValidTokenOp())
-	// token uses 1 second precision
-	time.Sleep(time.Second)
```

Comment:
Was this not important?

Reply:
I guess the intention was to generate a token that would expire "now" (reflected by `op.Duration = 0`) so a `time.Sleep` will guarantee the expiration (which won't work because `GenerateLoginToken` replaces the `0` value). Now I just `op.Duration = -1` so it's already expired "1 second ago".

P.S. Even though it expires "now", a sleep is not necessary because `ValidateToken` requires `now < expiry`.